api/placer.py

```diff
@@ -161,13 +161,22 @@ def process_file_field(self, field, info):
         container   = target['container']
         r_metadata  = target['metadata']
 
-        self.container_type = container.level
-        self.id_            = container.id_
-        self.container      = container.container
-
         info.update(r_metadata)
 
-        self.save_file(field, info)
+        if container.level != 'subject':
+            self.container_type = container.level
+            self.id_            = container.id_
+            self.container      = container.container
+            self.save_file(field, info)
+        else:
+            if field is not None:
+                files.move_form_file_field_into_cas(field)
+            if info is not None:
+                container.upsert_file(info)
```

Comment:
I'm doing a modified version of that logic to handle files at the subject level.